JeffersonMoran/CovidConsume
# feature/starting-project

* feat: adding the cards layout
* feat: covid api consume
* feat: showing the 5 countries with most cases
* feat: showing the 5 countries with fewer cases

how to test:
       npm install
       npm start
       
       you should be able to see the features in the home page.
